marlowe/doc/marlowe-model.adoc

```diff
@@ -17,7 +17,7 @@ We should separate the notions of _participants_ and _roles_ in a Marlowe contra
 
 ==== Accounts
 
-The Marlowe model allows for a contract to control money in a number of disjoint _accounts_: this allows for more explicit control of how the money flows in the contract. Each account is owned by a particular party to the contract, and that party receives a _refund_ of any remaining funds in the account when the contract is closed.footnote:[A fully “fungible” model is supported by a contract with a single account, and with explicit allocation of any remaining funds on contract termination.] These accounts are _local_, in that they only exist as during the execution of the contract, and during that time they are only accessible by parties to the contract.
+The Marlowe model allows for a contract to control money in a number of disjoint _accounts_: this allows for more explicit control of how the money flows in the contract. Each account is owned by a particular party to the contract, and that party receives a _refund_ of any remaining funds in the account when the contract is closed. These accounts are _local_, in that they only exist as during the execution of the contract, and during that time they are only accessible by parties to the contract.
```

Comment:
I would change the `a number of disjoint accounts` since it is now unnecessarily vague. My sugestion:

> The Marlowe model allows for a contract to store assets. All parties that participate in the contract implicitly own an account with their name. All assets stored in the contract must be in the account of one of the parties, this way, when the contract is `Close`d, all assets that remain in the contract are refunded automatically to the owner of the account were they are stored. These accounts are _local_, in that they only exist during the execution of the contract, and during that time they are only accessible by parties to the contract.